backend/server/src/main/java/com/issuetracker/domain/IssuesNumber.java

```diff
@@ -0,0 +1,14 @@
+package com.issuetracker.domain;
+
+public class IssuesNumber {
+
+    private final Long[] issueNumber;
+
+    public IssuesNumber(Long[] issueNumber) {
+        this.issueNumber = issueNumber;
+    }
+
+    public Long[] getIssueNumber() {
+        return issueNumber;
+    }
```

Comment:
```suggestion
    public Long[] toList() {
        return issueNumber;
    }
```

일급 컬렉션의 경우에는 toList 로 통일하는 것은 어떨지 제안해봅니다.

Reply:
통일 완료했습니다!